### DISCUSSION TITLE
Support object literal property value shorthand, so you can write `{x,y}` instead of `{x:x,y:y}`

### DISCUSSION THREAD

**Discussion by josdejong** [josdejong/mathjs#2953] [Ideas]
It would be very convenient to have support for object literal property value shorthand in the expression parser like JavaScript has since ES6, so you can write `{x,y}` instead of `{x:x,y:y}`.

See: https://attacomsian.com/blog/javascript-object-property-shorthand

**Comment by dvd101x** [comment]
It would be very nice to have that capability.

**Comment by dvd101x** [comment]
Hi Jos,

I was giving some more thought.

Do you think the shorthand might be more useful if mathjs had scopes? (not a single scope)
Maybe it's independent of that, just checking with you.

I've seen there are some plans for multi line functions and control flow in which case maybe scopes would be nice. #2701

**Reply by josdejong** [reply]
I think this feature is not related to scopes, it is just a more compact way to write an object.

**Reply by dvd101x** [reply]
Thanks!

Now I see your point, and I agree it's unrelated to scopes.

My opinion was due to the way I was introduced to the shorthand in javascript.

**Comment by gwhitney** [comment]
Closing as duplicate of #2889.
